export const getTranscations=async (req,res)=>{
    try {
        
    } catch (error) {
        console.log(error);
        res.status(500).json({
            status:"failed",
            message:error.message
        })
        
        
    }
}
export const getDashboardInformation=async (req,res)=>{
    try {
        
    } catch (error) {
        console.log(error);
        res.status(500).json({
            status:"failed",
            message:error.message
        })
        
        
    }
}
export const addTransaction=async (req,res)=>{
    try {
        
    } catch (error) {
        console.log(error);
        res.status(500).json({
            status:"failed",
            message:error.message
        })
        
        
    }
}
export const transferMoneyToAccount=async (req,res)=>{
    try {
        
    } catch (error) {
        console.log(error);
        res.status(500).json({
            status:"failed",
            message:error.message
        })
        
        
    }
}